static/src/js/components/CustomDateTimeWidget/CustomDateTimeWidget.jsx

```diff
@@ -89,14 +93,21 @@ const CustomDateTimeWidget = ({
   }
 
   const handleChange = (newDate) => {
-    // The picket widget has a bug where it is not setting millis to 0 when selecting a time.
-    newDate.setMilliseconds(0)
+    // The picket widget has a bug where it is not setting millis to 0 when selecting a time,
+    // so we need to reset the millisecond part. When the date time string is pasted in or a value
```

Comment:
maybe, "the picker widget has a bug where it is not setting the ms. to 0 when a user selects a time.   We are working around this bug by setting the ms to 0 only if the user selects/chooses a time from the widget.   If they type in a date/time or paste in a date/time should , we should not perform this reset as we should keep whatever ms they user enters."

Reply:
Yes, updated.